FRA_WE_BACK/core/utils.py

```diff
@@ -0,0 +1,25 @@
+import jwt
+
+from django.http 	        import JsonResponse    
+
+from FRA_WE_BACK.settings     import SECRET_KEY                             
+from users.models             import User
+
+
+def LogInDecorator(func):
+    def wrapper(self, request, *args, **kwargs):
+        try:
+            access_token = request.headers.get('Authorization', None)          
+            payload      = jwt.decode(access_token, SECRET_KEY, algorithms='HS256')  
+            user   	     = User.objects.get(id=payload['id'])                 
+            request.user = user
+
+        except jwt.exceptions.DecodeError:                                        
+            return JsonResponse({'message' : 'INVALID_TOKEN', 'Token':access_token }, status=400)
```

Comment:
DecodeError에 token을 전달해주는 이유가 있을까요?
```suggestion
            return JsonResponse({'message' : 'INVALID_TOKEN'}, status=400)
```